### DISCUSSION TITLE
Option or window rule to focus workspace

### DISCUSSION THREAD

**Discussion by Nairou** [YaLTeR/niri#396] [Ideas]
I'd like the option, either global setting or per-window-rule, to force a `focus-workspace` when opening a new window on a workspace that isn't visible.

I have multiple rules to open certain programs on different workspaces, but there is no visual indication that they were opened. After launching the program, I have to manually switch workspaces. I can see this being useful for some, but I'd like the option of it being automatic.

**Comment by YaLTeR** [comment]
I guess it could make sense. Probably still without stealing the focus away from a focused fullscreen window though.

**Reply by Nairou** [reply]
A very reasonable condition!
